pkg/snapstore/s3_snapstore_test.go

```diff
@@ -207,6 +205,95 @@ func (m *mockS3Client) ListObjectsPages(in *s3.ListObjectsInput, callback func(*
 	return nil
 }
 
+// ListObjectVersionsPages returns the versioned objects from map for mock test.
+func (m *mockS3Client) ListObjectVersionsPages(in *s3.ListObjectVersionsInput, callback func(*s3.ListObjectVersionsOutput, bool) bool) error {
+	var (
+		count    int64 = 0
+		limit    int64 = 1 // aws default is 1000.
+		lastPage bool  = false
+		keys     []string
+		out      = &s3.ListObjectVersionsOutput{
+			Prefix:   in.Prefix,
+			Versions: make([]*s3.ObjectVersion, 0),
+		}
+	)
```

Comment:
```suggestion
	var (
		count    int64
		limit    int64 = 1 // aws default is 1000.
		lastPage bool
		keys     []string
		out      = &s3.ListObjectVersionsOutput{
			Prefix:   in.Prefix,
			Versions: make([]*s3.ObjectVersion, 0),
		}
	)
```
Just a nit as the zero values are what you wanted only.

Reply:
pushed the changes.